Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -364,10 +367,8 @@ class BrowserViewController: UIViewController {
         }
 
         webViewContainerToolbar.snp_makeConstraints { make in
-            make.trailing.equalTo(webViewContainer)
-            make.leading.equalTo(webViewContainer)
+            make.leading.trailing.top.equalTo(webViewContainer)
```

Comment:
Nit: Is there a reason for using leading/trailing instead of left/right? The only thing I can think of is layout changes for RTL text but I don't think we need that for the web view container.